prowler/config/config.py

```diff
@@ -9,7 +9,7 @@
 
 timestamp = datetime.today()
 timestamp_utc = datetime.now(timezone.utc).replace(tzinfo=timezone.utc)
-prowler_version = "3.1.2"
+prowler_version = "3.1.1"
```

Comment:
Be careful with this.

```suggestion
prowler_version = "3.1.2"
```